manifest.txt

```diff
@@ -25,3 +25,4 @@ pylint 1.7.1
 requests 2.17.1
 responses 0.5.1
 protobuf 3.12.0
+oppia-ml-proto 0.0.0
```

Comment:
Are you planning to version oppia-ml-proto? It seems like a good idea given the protos will change, and their changes should be deliberately included once that side of the communication channel is ready to update.

Reply:
Yes, that is the plan. Oppia-ml-proto will be released and release versions will be used on Oppia and Oppia-ml to keep them in sync.